test/liblangutil/Scanner.cpp

```diff
@@ -819,6 +840,71 @@ BOOST_AUTO_TEST_CASE(irregular_line_breaks_in_strings)
 	}
 }
 
+BOOST_AUTO_TEST_CASE(solidity_keywords)
+{
+	// These are tokens which have a different meaning in Yul.
+	string keywords = "return byte bool address var in true false leave switch case default";
+	Scanner scanner(CharStream(keywords, ""));
+	BOOST_CHECK_EQUAL(scanner.currentToken(), Token::Return);
+	BOOST_CHECK_EQUAL(scanner.next(), Token::Byte);
+	BOOST_CHECK_EQUAL(scanner.next(), Token::Bool);
+	BOOST_CHECK_EQUAL(scanner.next(), Token::Address);
+	BOOST_CHECK_EQUAL(scanner.next(), Token::Var);
+	BOOST_CHECK_EQUAL(scanner.next(), Token::In);
+	BOOST_CHECK_EQUAL(scanner.next(), Token::TrueLiteral);
+	BOOST_CHECK_EQUAL(scanner.next(), Token::FalseLiteral);
+	BOOST_CHECK_EQUAL(scanner.next(), Token::Identifier);
+	BOOST_CHECK_EQUAL(scanner.next(), Token::Switch);
+	BOOST_CHECK_EQUAL(scanner.next(), Token::Case);
+	BOOST_CHECK_EQUAL(scanner.next(), Token::Default);
```

Comment:
Oh, I thought these 3 would be `Identifier` in Solidity mode.

Reply:
They are actual keywords, not yet used in the language.